examples/sdk/react/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@backtrace/react-example",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/react": "^13.4.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/jest": "^27.5.2",
+        "@types/node": "^16.18.37",
```

Comment:
types/node?

Reply:
This and all of the testing library stuff are just part of the default CRA typescript template. I've removed them now.